gradle/src/main/java/io/paketo/demo/DemoApplication.java

```diff
@@ -0,0 +1,20 @@
+package io.paketo.demo;
```

Comment:
pls change all the package name to "com.example" as the other test case

Reply:
OK, updated.